### PR TITLE
Making it easier to understand dependency review for public versus private repos

### PR DESCRIPTION
**Pull request by NickLiffen**
### Why:

Making it easier to understand where Dependency Review is available. This has caused some confusion. Hopefully, the proposed changes make it easier to understand but I welcome feedback 👍 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@NickLiffen 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by NickLiffen** [comment]
Thanks, @ramyaparimi 💯 I also welcome someone better at grammar than me to propose changes 

For example, another way of maybe phrasing this is:

```
Dependency review is available for all public repositories, as well as private repositories owned by organizations where ...
```

I very much welcome feedback on the proposed changes 👍

File: data/reusables/gated-features/dependency-review.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{% ifversion fpt or ghec %}Dependency review is available for all public repositories and for private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
+{% ifversion fpt or ghec %}Dependency review is available for all public repositories. As well as private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
```
Comment (mchammer01) [inline data/reusables/gated-features/dependency-review.md:1]: I think that the initial text is clear but I'm open to updates if this caused confusion with our customers. What about this?
(Will wait to hear from @jhutchings1 before applying this suggestion)
```suggestion
{% ifversion fpt or ghec %}Dependency review is available for all public repositories, as well as well as private repositories owned by organizations where {% data variables.product.prodname_GH_advanced_security %} is enabled.{% endif %}
```

**Review by mchammer01 — COMMENTED** [review]
@NickLiffen - thanks for suggesting an improvement. 
I've added a suggestion, let me know what you think 😃

File: data/reusables/gated-features/dependency-review.md (hunk 1 above)
Comment (NickLiffen) [reply data/reusables/gated-features/dependency-review.md:1]: yeah it caused some confusion, to be fair I even read it wrong 😮‍💨 I think the purpose is making it a little more apparent. I read this as you needed a GHAS licence to run this on both public repos and private repos. I think it makes sense to decouple it, like you have ❤️

Comment (jhutchings1) [reply data/reusables/gated-features/dependency-review.md:1]: I like @mchammer01 's suggestion above. The content is accurate in regards to feature availability, but if we can make it more clear, we should.

Comment (NickLiffen) [reply data/reusables/gated-features/dependency-review.md:1]: thanks, @jhutchings1 👍 @mchammer01 are we good to commit your suggestion, please? I think this pleases everyone ❤️ thank you all.

**Comment by mchammer01** [comment]
@NickLiffen - thank you for highlighting that this sentence was confusing. I hadn't realized until you mentioned it.
Will apply the suggestion and merge this PR.
Thanks again 💖

**Review by mchammer01 — APPROVED** [review]
LGTM ✨

**Pull request merged**